### PR TITLE
Add Labels.numpy and organize fixtures

### PR DESCRIPTION
**Pull request by roomrys**
### Description
Add a `Labels.numpy` method that converts a `Labels` object to a numpy. Also reorganize the fixtures and add the video for the centered_pair_predictions.slp.

### Types of changes

- [ ] Bugfix
- [x] New feature
- [ ] Refactor / Code style update (no logical changes)
- [ ] Build / CI changes
- [ ] Documentation Update
- [ ] Other (explain)

### Does this address any currently open issues?
[list open issues here]

### Outside contributors checklist

- [ ] Review the [guidelines for contributing](https://github.com/talmolab/sleap-io/blob/main/CONTRIBUTING.md) to this repository
- [ ] Add tests that prove your fix is effective or that your feature works
- [ ] Add necessary documentation (if appropriate)

#### Thank you for contributing to SLEAP-IO!
:heart:

### CODE REVIEW

File: sleap_io/model/labeled_frame.py [hunk 1]
```diff
@@ -59,5 +59,5 @@ def numpy(self) -> np.ndarray:
         n_nodes = len(self.instances[0]) if n_instances > 0 else 0
         pts = np.full((n_instances, n_nodes, 2), np.nan)
         for i, inst in enumerate(self.instances):
-            pts[i] = inst.numpy()
+            pts[i] = inst.numpy()[:, 0:2]
         return pts
```
Comment (roomrys) [inline sleap_io/model/labeled_frame.py:63]: Modified s.t. `labeled_frame.numpy` returns points only even if there are `PredictedInstances`. The other option is to make `labeled_frame.numpy` always return 3 coordinates with the last coordinate (the score) equal to `np.nan` if type is `Instance`. The current solution (no score returned) seems cleaner and more consistent to me personally.

File: sleap_io/model/labels.py [hunk 2]
```diff
@@ -88,3 +88,96 @@ def __repr__(self) -> str:
...
+            This method assumes that instances have tracks assigned and is intended to
+            function primarily for single-video prediction results.
+        """
+        # Get labeled frames for specified video.
+        if video is None:
+            video = 0
+        if type(video) == int:
+            video = self.videos[video]
+        lfs = [lf for lf in self.labeled_frames if lf.video == video]
+        print(f"video = {video}")
```
Comment (talmo) [inline sleap_io/model/labels.py:131]: Remove this print statement

**Review by talmo — CHANGES_REQUESTED** [review]
Minor fix

**Review by talmo — APPROVED** [review]
lgtm

**Pull request merged**